package.json

```diff
@@ -59,29 +59,29 @@
     "access": "public"
   },
   "devDependencies": {
-    "@types/jest": "^24.0.25",
-    "@typescript-eslint/eslint-plugin": "^2.15.0",
-    "@typescript-eslint/parser": "^2.15.0",
-    "codecov": "^3.6.1",
-    "codemirror": "^5.50.2",
+    "@types/jest": "^25.1.0",
+    "@typescript-eslint/eslint-plugin": "^2.18.0",
+    "@typescript-eslint/parser": "^2.18.0",
+    "codecov": "^3.6.2",
+    "codemirror": "^5.51.0",
     "eslint": "^6.8.0",
     "eslint-config-airbnb-base": "14.0.0",
-    "eslint-config-prettier": "^6.9.0",
+    "eslint-config-prettier": "^6.10.0",
     "eslint-plugin-import": "~2.18.2",
-    "jest": "^24.9.0",
+    "jest": "^25.1.0",
     "jest-junit": "^10.0.0",
     "lodash.debounce": "^4.0.8",
     "npm-run-all": "^4.1.5",
     "parcel": "^1.12.4",
     "prettier": "^1.19.1",
-    "rimraf": "^3.0.0",
-    "ts-jest": "^24.3.0",
-    "typescript": "^3.7.4"
+    "rimraf": "^3.0.1",
+    "ts-jest": "^25.0.0",
+    "typescript": "^3.7.5"
   },
   "dependencies": {
     "@slack/types": "^1.3.0",
     "he": "^1.2.0",
-    "htm": "^2.2.1",
+    "htm": "^3.0.2",
```

Comment:
`htm` v3.0.2 may have a wrong type definition that breaks TypeScript compile.
https://github.com/developit/htm/issues/152

Reply:
I'm trying to resolve in developit/htm#153.